roles/icingaweb2/tasks/manage_icingaweb_mysql_db.yml

```
@@ -0,0 +1,70 @@
+#- name: Manage DB resources
+#  ansible.builtin.include_tasks:
+
+- name: Check Database Credentials
+  ansible.builtin.assert:
+    that:
+      - icingaweb2_db['user'] is defined
+      - icingaweb2_db['password'] is defined
+    fail_msg: "No database credentials defined. Please set icingaweb2_db.<user|password> or a privileged user with icingaweb2_priv_db_<user|password>"
+  when: icingaweb2_priv_db_password is undefined and icingaweb2_priv_db_user is undefined
+
+- name: Set db user with admin privileges
+  ansible.builtin.set_fact:
+    _priv_db_user: "{{ icingaweb2_priv_db_user }}"
+    _priv_db_pass: "{{ icingaweb2_priv_db_password }}"
+  when: icingaweb2_priv_db_password is defined and icingaweb2_priv_db_user is defined
+
+- name: Ensure Icinga Web
+  block:
+    - name: Build mysql command
+      ansible.builtin.set_fact:
+        _tmp_mysqlcmd: >-
+          mysql {% if icingaweb2_db['host'] | default('localhost') != 'localhost'  %} -h "{{ icingaweb2_db['host'] }}" {%- endif %}
+          {% if icingaweb2_db['port'] is defined %} -P "{{ icingaweb2_db['port'] }}" {%- endif %}
+          {% if icingaweb2_db['ssl_mode'] is defined %} --ssl-mode "{{ icingaweb2_db['ssl_mode'] }}" {%- endif %}
+          {% if icingaweb2_db['ssl_ca'] is defined %} --ssl-ca "{{ icingaweb2_db['ssl_ca'] }}" {%- endif %}
+          {% if icingaweb2_db['ssl_cert'] is defined %} --ssl-cert "{{ icingaweb2_db['ssl_cert'] }}" {%- endif %}
+          {% if icingaweb2_db['ssl_key'] is defined %} --ssl-key "{{ icingaweb2_db['ssl_key']}}" {%- endif %}
```

```suggestion
          {% if icingaweb2_db['ssl_key'] is defined %} --ssl-key "{{ icingaweb2_db['ssl_key'] }}" {%- endif %}
```
This space will fix the warnings, I guess.